docs/source/usage/benchmarks.rst

```diff
@@ -0,0 +1,90 @@
+.. _usage-benchmark:
+
+Benchmarks
+==========
+
+Parallel benchmarks 8a & 8b
+---------------------------
+
+The following examples show parallel reading and writing of domain-decomposed data with MPI.
+
+The `Message Passing Interface (MPI) <https://www.mpi-forum.org/>`_ is an open communication standard for scientific computing.
+MPI is used on clusters, e.g. large-scale supercomputers, to communicate between nodes and provides parallel I/O primitives.
+
+Writing
+^^^^^^^
+
+*Source*: ``examples/8a_benchmark_write_parallel.cpp``
+
+This benchmark writes a few meshes and particles,
+either 1D, 2D or 3D.
+
+The meshes are viewed as grid of mini blocks.
+As an example, we assume the mini blocks dimension are [16, 32, 32].
+
+
+Next we define the grid based on the mini block.
+say, [32, 32, 16]. Then our actual mesh size is [16x32, 32x32, 32x16].
+
+
+Here is a sample input file:
```

Comment:
After reading this page a couple of times, I have to admit I do not know how to:
- call the executable(s)
- what arguments to pass
- how to name and where to place / how to pass the input file